### PR TITLE
Initial stub with license, CI, README.

### PR DESCRIPTION
**Pull request by waywardmonkeys**

### CODE REVIEW

**Comment by waywardmonkeys** [comment]
I should start this out with the cache fix I place.

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+[lints]
+clippy.doc_markdown = "warn"
+clippy.semicolon_if_nothing_returned = "warn"
+
+[features]
+
+[package.metadata.docs.rs]
+all-features = true
+
```
Comment (DJMcNab) [inline Cargo.toml:23]: We should specify a single target here. E.g. https://github.com/linebender/xilem/blob/c6c1b716405fa2c024e0d69eff22567d017f38aa/xilem_core/Cargo.toml#L25-L29

File: Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+repository = "https://github.com/linebender/interpoli"
+readme = "README.md"
+# Keep in sync with RUST_MIN_VER in .github/workflows/ci.yml, with the relevant README.md files
+# and with the MSRV in the `Unreleased` section of CHANGELOG.md.
+rust-version = "1.75"
+
+[lints]
+clippy.doc_markdown = "warn"
+clippy.semicolon_if_nothing_returned = "warn"
+
```
Comment (DJMcNab) [inline Cargo.toml:18]: Should we have this here? https://github.com/linebender/peniko/blob/3d040a9d4f309c2dbefbd96b64bfb0cc629fec33/Cargo.toml#L43-L111

**Review by DJMcNab — COMMENTED** [review]
Few nits, and a big thing about the lints table

File: Cargo.toml (hunk 1 above)
Comment (waywardmonkeys) [reply Cargo.toml:23]: I don't think that's necessary, but I've added it.

File: Cargo.toml (hunk 2 above)
Comment (waywardmonkeys) [reply Cargo.toml]: I've added the non-commented out parts and not `clippy.cargo`.

File: Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+[features]
+
```
[the remark this answers is not in the document]
Comment (waywardmonkeys) [reply Cargo.toml:19]: The `libm` feature is needed for the CI. But we will use it later to enable libm in kurbo / peniko.

We don't (yet) have a dep on `libm` here to add.

File: README.md [hunk 4]
```diff
@@ -1 +1,67 @@
-# interpoli
\ No newline at end of file
+<div align="center">
+
+# Interpoli
+
+**A Rust 2D graphics type library**
```
Comment (DJMcNab) [inline README.md:5]: I think this was incorrectly copied

File: Cargo.toml (hunk 1 above)
Comment (DJMcNab) [reply Cargo.toml:23]: The important part of the line is more:
```suggestion
# Interpoli is entirely platform-agnostic, so only display docs for one platform
targets = []

```

But it's also fine not to have.

File: Cargo.toml (hunk 3 above)
[the remark this answers is not in the document]
Comment (DJMcNab) [reply Cargo.toml:19]: :+1:

File: src/lib.rs [hunk 5]
```diff
@@ -0,0 +1,6 @@
+// Copyright 2024 the Interpoli Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#![cfg_attr(all(not(feature = "std"), not(test)), no_std)]
+
```
Comment (DJMcNab) [inline src/lib.rs:5]:
```suggestion
#![warn(unused_crate_dependencies)]

```

**Review by DJMcNab — APPROVED** [review]
Looks good!

**Pull request merged**